tests/index.js

```diff
@@ -38,4 +40,46 @@ tap.test('should not compile schema with unknown keywords', async t => {
         await fastify.close()
         t.pass()
     }
+})
+
+tap.test('should validate querystring parameters', async t => {
+    t.plan(1)
+    const fastify = Fastify().setValidatorCompiler(TypeBoxValidatorCompiler).get('/', {
+        schema: {
+            querystring: Type.Object({
+                a: Type.String(),
+                b: Type.String(),
+                c: Type.String()
+            })
+        }
+    }, (req, res) => res.send(req.query))
+    await fastify.listen({ port: 5000 })
```

Comment:
You can use `fastify.inject` instead of `node-fetch`.

Reply:
It's actually better to use .inject() in this case.

Reply:
Amended